DESCRIPTION

```diff
@@ -40,6 +41,7 @@ URL: https://github.com/pik-piam/mrremind
 Depends:
     R (>= 4.0)
 Imports:
+    abind,
```

Comment:
Are you sure you need to import a new library for what you want to achieve. Most likely there is a madrat-way

Reply:
Ok, works for me, then.